server/sonar-server/src/main/java/org/sonar/server/computation/source/LastCommitVisitor.java

```diff
@@ -78,6 +78,10 @@ public void visitFile(Component file, Path<LastCommit> path) {
     // load SCM blame information from report. It can be absent when the file was not touched
     // since previous analysis (optimization to decrease execution of blame commands). In this case
     // the date is loaded from database, as it did not change from previous analysis.
+
+    // TODO We should use ScmInfoRepository instead of reading the report
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)